Samples/Swift/DaysUntilBirthday/Shared/Services/GoogleSignInAuthenticator.swift

```diff
@@ -35,12 +35,28 @@ final class GoogleSignInAuthenticator: ObservableObject {
       print("There is no root view controller!")
       return
     }
-
-    GIDSignIn.sharedInstance.signIn(withPresenting: rootViewController) { signInResult, error in
+    let manualNonce = UUID().uuidString
+
+    GIDSignIn.sharedInstance.signIn(
+      withPresenting: rootViewController,
+      hint: nil,
+      additionalScopes: nil,
+      nonce: manualNonce
+    ) { signInResult, error in
       guard let signInResult = signInResult else {
         print("Error! \(String(describing: error))")
         return
       }
+
+      // Per OpenID Connect Core section 3.1.3.7, rule #11, compare returned nonce to manual
+      guard let idToken = signInResult.user.idToken?.tokenString,
```

Comment:
with the change in this PR, AppAuth is does this validation, so I think there could be a note saying that this is not strictly necessary, but recommend?

Reply:
While AppAuth does the validation, I think it still makes sense to provide an example since AppAuth is an intermediate dependency here (from the client's point of view). If I were an app developer using these libraries, I would want to verify that the nonce is what I expect.

Reply:
Sounds good!👍